components.json

```diff
@@ -925,6 +925,11 @@
 		"yaml": {
 			"title": "YAML",
 			"owner": "hason"
+		},
+		"cmake": {
```

Comment:
The language list is largely sorted alphabetically, so please insert this before CoffeeScript.